guide/payments/request.md

```diff
@@ -0,0 +1,471 @@
+   layout = "shadow"
+   caption = "Reusable payment requests, like Lightning addresses, are useful for receiving recurring payments."
+   alt-text = "Screen showing reusable Lightning address."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/withdraw-request.png"
+   retina = "/assets/images/guide/payments/request/withdraw-request@2x.png"
+   layout = "shadow"
+   caption = "A withdraw request is another way a user can request bitcoin."
+   alt-text = "Screen showing withdraw request UI."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Single-use payment requests
+
+The most common way bitcoin is requested today is in the form of single-use payment requests. For Lightning, a standard [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoice is used. For on-chain, [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}) are used. Addresses can be reused though this is not advised due to privacy concerns. single-use invoices and addresses need to be easily and conveniently generated within a wallet application.
+
+### Amount
+
+When generating a single-use payment request, users need to be able to add an amount to be paid.  Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
+
+Amount units need to be toggleable between bitcoin, satoshi and local currency values as senders may request in a set unit. Ensure the unit being entered is prominent to prevent entering of the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
+
+### Address types
+
+There are several address types that can be used for on-chain transfer requests. Each address type has unique benefits, such as SegWit saving on network fees. Supporting every address type makes a wallet more interoperable and lets users access the benefits offered by each type. More on [address types]({{ "/guide/glossary/address/" | relative_url }}) page.
+
+### Meta data
+
+A payment request with only standard data like an amount and date communicates little to both parties about what the payment was for. Users should have the option to attach a note for record keeping and/or for the sender to read. Adding tags, labels and any other metadata that may be relevant to give more context to the payment is also beneficial.
+
+#### Invoice expirations
+
+A piece of meta data unique to invoices is an expiration time. This is how long until a single-use invoice expires. By default this is 60 minutes. When an invoice is generated an expiration timer needs to be shown to users alongside the payment request. Advanced users may want the ability to adjust this expiration time for different use cases.
```

Comment:
Personally, I understand what you meant here, but a reader may be confused a bit why `invoice` terminology is all of a sudden used when you talked about `payment request`. In context of single-use payment requests there's no timer for an on-chain, so I'm not sure how relevant this section is.

Reply:
I don’t believe the LN spec defaults to 60 minutes so this could be more accurately stated, I think.

Reply:
It's mentioned what an invoice is in the first paragraph of the Single-use payment requests section. I can try address it in another way but I think discussing expiration times is important a touched in the [slack discussion](https://bitcoindesign.slack.com/archives/C015856BDME/p1641858282015200) we had the other day.